### PR TITLE
Translation to pt-br

### PR DESCRIPTION
**Pull request by kennedyoliveira**

### CODE REVIEW

**Comment by vania-pooh** [comment]
@kennedyoliveira thank you for contribution!
@just-boris please merge if all this stuff with plurals and so on seems to be correct.

**Comment by kennedyoliveira** [comment]
@vania-pooh Your Welcome! If something is wrong, please let me know that i'll fix it!

**Comment by just-boris** [comment]
@kennedyoliveira great!

I made a little redesign of language switcher in https://github.com/allure-framework/allure-core/pull/614

Please rebase you branch onto latest master and look how it looks now

File: allure-report-face/src/main/webapp/js/controllers.js [hunk 1]
```diff
@@ -51,6 +51,9 @@ angular.module('allure.core.controllers', [])
         }, {
             name: "РУС",
             locale: "ru"
+        },{
+           name: "PT-BR",
```
Comment (just-boris) [inline allure-report-face/src/main/webapp/js/controllers.js]: Use full name here.
I've changed this field for other languages in `master` branch

**Comment by baev** [comment]
@kennedyoliveira thanks for all the work. Could you please fix comment and lets merge this

**Comment by kennedyoliveira** [comment]
@baev done!

**Comment by just-boris** [comment]
image
LGTM

**Comment by kennedyoliveira** [comment]
@just-boris what does LGTM means?

**Comment by kennedyoliveira** [comment]
@just-boris never mind, I'm glad to help!

**Comment by just-boris** [comment]
LGTM (looks good to me) is a message for @baev that I accept this changes

**Comment by baev** [comment]
@kennedyoliveira could you please squash your commits into one. See http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request and http://stackoverflow.com/questions/14534397/how-to-squash-all-my-commits-into-one-github

**Comment by kennedyoliveira** [comment]
@baev done!

**Comment by baev** [comment]
@kennedyoliveira thanks for all the work!

**Comment by kennedyoliveira** [comment]
@baev You are welcome!

**Comment by baev** [comment]
@kennedyoliveira do you mind to contribute pt-br translation to Allure 2?

**Comment by kennedyoliveira** [comment]
Hey @baev, Sorry i Just saw your comment today, sure i can help! I did tried the version 2 yet, but Will take a look and translate !

**Pull request merged**